src/qt/trafficgraphwidget.cpp

```diff
@@ -128,8 +128,9 @@ void TrafficGraphWidget::updateRates()
 
     quint64 bytesIn = clientModel->node().getTotalBytesRecv(),
             bytesOut = clientModel->node().getTotalBytesSent();
-    float inRate = (bytesIn - nLastBytesIn) / 1024.0f * 1000 / timer->interval();
-    float outRate = (bytesOut - nLastBytesOut) / 1024.0f * 1000 / timer->interval();
+    // Rates are in bytes/millisecond which is equal to kilobytes/second
```

Comment:
Instead of adding a comment, is it better just choose proper names for variables?
E.g., `in_rate_kilobytes_per_sec` ?